src/connectors/hits-per-page/__tests__/connectHitsPerPage-test.js

```diff
@@ -234,10 +242,13 @@ See documentation: https://www.algolia.com/doc/api-reference/widgets/hits-per-pa
     });
 
     expect(
-      widget.getConfiguration(new SearchParameters({ hitsPerPage: 20 }))
+      widget.getWidgetSearchParameters(
+        new SearchParameters({ hitsPerPage: 20 }),
+        { uiState: {} }
+      )
     ).toEqual(
       new SearchParameters({
-        hitsPerPage: 20,
+        hitsPerPage: 10,
```

Comment:
isn't this supposed to be 20?
this doesn't look like it overrides it, right?

Reply:
we override the previously set hPP with the "default" one, this was defined by #4069